inc/gutenberg-block-edits.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+/**
+ * Enqueue scripts to extend default block editor.
+ */
+function largo_extend_block_editor() {
+
+    wp_enqueue_script(
+        'blocks-image-customizations',
+		get_template_directory_uri() . '/js/blocks-image-customization.js',
+        array( 'wp-blocks', 'wp-i18n', 'wp-element', 'wp-editor', 'wp-api' ),
+        '1.0.0',
+        true
+    );
+	
+}
+add_action( 'enqueue_block_editor_assets', 'largo_extend_block_editor' );
+
+/**
+ * Register custom fields for the REST api
+ */
+function largo_register_custom_rest_fields() {
+
+	register_rest_field( 'attachment', 'media_credit', 
+		array(
+			'get_callback' => 'largo_display_media_credit_in_rest_api',
+			'schema' => null,
+		)
+	);
+
+}
+add_action( 'rest_api_init', 'largo_register_custom_rest_fields' );
```

Comment:
This should probably go in the file where we register that metadata to begin with.